lib/src/components/NxTag/NxTag.tsx

```diff
@@ -25,10 +25,10 @@ const NxTag: FunctionComponent<Props> =
 
       return (
         <NxOverflowTooltip>
-          <div className={tagClasses} {...attrs}>
+          <label className={tagClasses} {...attrs}>
```

Comment:
I wonder if this could be a breaking change:
In the doc it mentioned that it accepts HTML div attributes: https://gallery.sonatype.dev/#/pages/NxTag
I wonder if wrapping the label in div and moving the classname and attrs back to the div might fix it.

Reply:
> I wonder if this could be a breaking change:
In the doc it mentioned that it accepts HTML div attributes: https://gallery.sonatype.dev/#/pages/NxTag
I wonder if wrapping the label inside div and moving the classname and attrs back to the div might fix it.

I suspect `div` attrs are a subset of `label` attrs.  If that's true this wouldn't be a breaking change.

Reply:
> > I wonder if this could be a breaking change:
> > In the doc it mentioned that it accepts HTML div attributes: https://gallery.sonatype.dev/#/pages/NxTag
> > I wonder if wrapping the label inside div and moving the classname and attrs back to the div might fix it.
> 
> I suspect `div` attrs are a subset of `label` attrs. If that's true this wouldn't be a breaking change.

The only attributes of `div` are the global attributes. the `label` also includes the global attributes.